Banner strikes

YannickdeZwart opened the pull request:
Added extra Message option to guildEvents.json on timestamp1, timestamp2, filler
Gives a strike when exeeding maxBannerAmount for timestamp

![afbeelding](https://github.com/user-attachments/assets/007b1711-cf6f-4e9d-bc67-457431be458e)

rohid2000 commented on json/guildEvents.json:
Would be best to put the output of (bannerStrikes) in the strike-list Excel file when the strike-list Excel would be send after Attack Phase of TW is live.

rohid2000 reviewed (changes requested):
Looks good, requested for 1 adjustment

YannickdeZwart commented:
added Feature:
 - added events in database that get linked with specific strikes
 - excel has different worksheets for each strike type (can be read in also /strike sync excel)
   - you can see for which event you are giving strikes on the top right in the excel
   - every player with a string in the reason tab gets a strike with that reason
 - removed strike cleanup for better strike tracking

rohid2000 reviewed (approved):
Looks good!
